src/handlers/post-pessoas.cc

```diff
@@ -0,0 +1,180 @@
+#include <drogon/drogon.h>
+
+using namespace std;
+using namespace drogon;
+
+auto postPessoas()
+{
+  return [](const HttpRequestPtr &req,
+            function<void(const HttpResponsePtr &)> &&callback)
+  {
+    auto jsonPtr = req->jsonObject();
+    if (jsonPtr == nullptr)
+    {
+      callback(makeFailedResponse());
+      return;
+    }
+
+    auto nickname = jsonPtr->get("apelido", "").as<string>();
+    auto name = jsonPtr->get("nome", "").as<string>();
+    auto birthDate = jsonPtr->get("nascimento", "").as<string>();
+    auto stacksJson = jsonPtr->get("stack", "");
+
+    if (nickname.size() == 0)
+    {
+      callback(makeUnprocessableContentResponseResponse("The 'apelido' parameter should not be empty"));
+      return;
+    }
+
+    if (name.size() == 0)
+    {
+      callback(makeUnprocessableContentResponseResponse("The 'nome' parameter should not be empty"));
+      return;
+    }
+
+    if (birthDate.size() == 0)
+    {
+      callback(makeUnprocessableContentResponseResponse("The 'nascimento' parameter should not be empty"));
+      return;
+    }
+
+    if (stacksJson.size() == 0)
```

Comment:
One of the problems you have issued on X is here. The `stack` parameter is not required.